sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/events/RecognizeCompleted.java

```diff
@@ -60,4 +61,19 @@ public CollectTonesResult getCollectTonesResult() {
     public ChoiceResult getChoiceResult() {
         return this.choiceResult;
     }
+
+    /**
+     * Get the collectToneResult or choiceResult property.
+     *
+     * @return the recognizeResult value.
+     */
+    public RecognizeResult getRecognizeResult() {
+        if (this.recognitionType == CallMediaRecognitionType.DTMF) {
+            return this.collectTonesResult;
+
+        } else if (this.recognitionType == CallMediaRecognitionType.CHOICES) {
+            return this.choiceResult;
+        }
+        return null;
```

Comment:
what about returning an `Optional<RecognizeResult>` instead and omit returning null?

Reply:
good point, will change it to optional